superbench/runner/runner.py

```diff
@@ -58,6 +58,27 @@ def __get_enabled_benchmarks(self):
                 return list(self._sb_config.superbench.enable)
         return [k for k, v in self._sb_benchmarks.items() if v.enable]
 
+    def __get_mode_command(self, mode, exec_command):
+        """Get runner command for given mode.
+
+        Args:
+            mode (str): Runner mode.
+            exec_command (str): Executor command.
+
+        Return:
+            str: Runner command.
+        """
+        if mode == 'torch.distributed':
+            # TODO: replace with torch.distributed.run in v1.9
+            return (
+                'python3 -m torch.distributed.launch '
+                '--use_env --no_python --nproc_per_node=8 '
```

Comment:
Is it hard code for `--nproc_per_node`?

Reply:
currently yes, will change to `--nproc_per_node=gpu` in torch 1.9, here's [reference](https://github.com/pytorch/pytorch/blob/b4f3a989da375675785754e4b12d2ce757a1ffaa/torch/distributed/run.py#L269-L273)

if we want to support dynamic number, e.g. run distributed training on 4 gpus even if there're 8, we may need to add an option in config file.

Reply:
Yes, we definitely need to support it.

Reply:
added